client/app/bundles/HelloWorld/components/dashboard/my_classes.jsx

```diff
@@ -1,9 +1,8 @@
 'use strict'
 
  import React from 'react'
- import AddClassMini from './add_class_mini.jsx'
  import ClassMini from './class_mini.jsx'
- import SyncGoogleClassroomsMini from './sync_google_classrooms_mini.jsx'
+ import AddOrSyncClassroomsMini from './add_or_sync_classrooms_mini.jsx'
```

Comment:
@RyanNovas This line is causing an issue as the file doesn't exist. Is it supposed to be sync_google_classrooms_mini.jsx ?